tests/unit_tests/cpu/energy_loss.cpp

```diff
@@ -151,6 +151,55 @@ INSTANTIATE_TEST_SUITE_P(
     ::testing::Values(std::make_tuple(silicon<scalar>(),
                                       100.1f * unit<scalar>::GeV, 2.451f)));
 
+INSTANTIATE_TEST_SUITE_P(
+    Bethe_0p1GeV_Fe, EnergyLossBetheValidation,
+    ::testing::Values(std::make_tuple(iron<scalar>(),
+                                      0.1003f * unit<scalar>::GeV, 2.274f)));
+
+/*
```

Comment:
Are these not working?

Reply:
No they are not. I wrote the discrepancy together